### PR TITLE
Fix/unify the look with the web app

### PR DESCRIPTION
**Pull request by StanislavPrime**

### CODE REVIEW

File: examples/reminders/lib/feature_reminder_manage/blocs/reminder_manage_bloc.dart [hunk 1]
```diff
@@ -8,15 +8,20 @@ import '../../base/services/reminders_service.dart';
...
+part 'reminder_manage_bloc_extensions.dart';
+
 /// A contract class containing all events of the ReminderManageBloC.
 abstract class ReminderManageBlocEvents {
   void update(ReminderModel reminder);
 
+  @RxBlocEvent(type: RxBlocEventType.behaviour, seed: '')
+  void setName(String title);
+
+  void validate();
```
Comment (StanislavPrime) [inline examples/reminders/lib/feature_reminder_manage/blocs/reminder_manage_bloc.dart:20]: Remove the `validate()` method

File: examples/reminders/lib/feature_reminder_manage/blocs/reminder_manage_bloc.dart [hunk 2]
```diff
@@ -34,14 +39,19 @@ abstract class ReminderManageBlocStates {
...
   ConnectableStream<Result<IdentifiablePair<ReminderModel>>> get onUpdated;
 
-  ConnectableStream<Result<ReminderModel>> get onCreated;
+  Stream<Result<ReminderModel>> get onCreated;
+
+  Stream<String> get name;
+
+  Stream<String?> get nameErrorMessage;
+
+  Stream<bool> get isNameValid;
```
Comment (StanislavPrime) [inline examples/reminders/lib/feature_reminder_manage/blocs/reminder_manage_bloc.dart:48]: Change the type of the `onCreated` stream to `ConnectableStream`.
Remove the `nameErrorMessage` state.
Remove the `isNameValid ` state.

File: examples/reminders/lib/feature_reminder_manage/blocs/reminder_manage_bloc.dart (hunk 1 above)
Comment (StanislavPrime) [reply examples/reminders/lib/feature_reminder_manage/blocs/reminder_manage_bloc.dart]: The method was removed

File: examples/reminders/lib/feature_splash/splash_page.dart [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+        body: Center(
+          child: SizedBox(
+            height: 200,
+            child: Column(
+              children: [
+                _checkIsUserLoggedIn(),
```
Comment (StanevPrime) [inline examples/reminders/lib/feature_splash/splash_page.dart:42]: @StanislavPrime, be aware that according to the best practices the widget (in this case `RxBlocListener`) should be placed directly in the build method, while its callback (in this case the `listener`) should be extracted as a private method.

**Review by StanevPrime — APPROVED** [review]

File: examples/reminders/lib/feature_splash/splash_page.dart (hunk 3 above)
Comment (StanislavPrime) [reply examples/reminders/lib/feature_splash/splash_page.dart]: I have placed the `RxBlocListener` in the build method and extracted the `listener` callback as a private method.

**Pull request merged**